### DISCUSSION TITLE
How to remove the kitten image when running sdpsettings statement

### DISCUSSION THREAD

**Discussion by Pei-w** [yalmip/YALMIP#899] [Problems using YALMIP] [answered]
Which cat picture is very cute, I also like it very much, but the program I run needs to loop d to call the `sdpsettings` in yalmip. Is there any way for a friend to help me **remove the cat picture temporarily**? If this question can be answered, I will be very grateful

**Comment by johanlofberg** [comment, marked as answer]
sdpsettings is not in any sense related to the abused kitten.

The only sensible way to remove the cat is to fix the underlying problem with strict inequalities

https://yalmip.github.io/inside/strictinequalities/

...or you install the develop version where the cat is gone, but that's because strict inequalities are removed completely from the language

**Comment by johanlofberg** [comment]
Btw, as this is not an issue with the code but more of a usage question, I've converted this to a thread in the discussions instead

**Reply by Pei-w** [reply]
Hello, I'm sorry because this problem takes up your time. I use strict inequality when dealing with the problem. I'll download a development version and try it. I'm glad you can reply to me. Thank you.

**Reply by johanlofberg** [reply]
Just so you understand: The develop version will not allow you to use strict inequalities and thus remove the kitten. In the develop version, your code will simply not run with strict inequalities, so there is no need to show the crying kitten.
